custom-pages/universal-login/src/components/Register/Register.js

```diff
@@ -168,10 +168,17 @@ class Register extends Component {
     });
   };
 
-  goToAlert = (event) => {
-    if (event) event.preventDefault();
+  goToAlert = (errorName, event) => {
+    event.preventDefault();
 
-    getFirstErrorElement(this.state.errors).scrollIntoView({ block: 'center' })
+    document.getElementsByName(errorName)[0].scrollIntoView({ block: 'center' });
+  };
+
+  removeRequiredErrorPrefix = (errorMessage, requiredMessage) => {
```

Comment:
i'm not sure what this function does based on it's name. i.e. what's an error prefix?

i think it might just need a better name, if that's ok?

Reply:
Yeah, it is a little confusing. I put the field name at the beginning of the error messages, so it was clear what the error was for (in the alert box). This function then removes the field name from the beginning. Hence it transforms 'Password - This field is required' into 'This field is required'.

Reply:
I will think of a better name.